openedx/core/djangoapps/content_libraries/api/containers.py

```diff
@@ -60,7 +61,55 @@
 
 
 class ContainerType(Enum):
+    """
+    The container types supported by content_libraries, and logic to map them to OLX.
+    """
     Unit = "unit"
+    Subsection = "subsection"
+    Section = "section"
+
+    @property
+    def olx_tag(self) -> str:
```

Comment:
I'm torn between calling these block_types vs olx_tags. On one hand, we generally don't want to think of Containers as pseudo-XBlocks, so I'm trying to frame this all as an OLX thing. On the other hand, container_sync will need to understand that Unit maps to the block_type `vertical`.